### DISCUSSION TITLE
Power Platform Command Line Interface: July Update

### DISCUSSION THREAD

**Discussion by petrochuk** [microsoft/powerplatform-build-tools#493] [Announcements]
Blog: https://devblogs.microsoft.com/powerplatform/power-platform-command-line-interface-july-update/

Release notes: https://www.nuget.org/packages/Microsoft.PowerApps.CLI#release-body-tab

**Comment by fowl2** [comment]
Any chance of the CLI becoming open source? It'd be really nice to be able to contribute!
